server/package-lock.json

```diff
@@ -11,7 +11,7 @@
       "dependencies": {
         "@aws-sdk/client-iot": "^3.360.0",
         "@aws-sdk/client-iot-wireless": "^3.360.0",
-        "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v10.2.0",
```

Comment:
This is sort of strange. I was expecting there to be more changes in the `package-lock.json` file. I would have expected further down there to be a change in the git commit hash for the latest commit on your `ad-feature/Spanish_translation` branch.

Could you please try removing this dependency, adding it again, and then re-running `npm install` and see if you get some more differences in this file? Thanks!